MismatchGeneration/1_mismatch_generations/finding-mismatches-fide.ipynb

```diff
@@ -0,0 +1,702 @@
+      "WIKIDATA: http://www.wikidata.org/entity/Q104630109\n",
+      "Birthday not found on FIDE for FIDE id 4109058\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q104639267\n",
+      "Birthday not found on FIDE for FIDE id 4109414\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q104701623\n",
+      "Birthday not found on FIDE for FIDE id 4108710\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q104760644\n",
+      "Birthday not found on FIDE for FIDE id 4108175\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q104872493\n",
+      "Birthday not found on FIDE for FIDE id 402567\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q106255695\n",
+      "Birthday not found on FIDE for FIDE id 9800190\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q106636448\n",
+      "Birthday not found on FIDE for FIDE id 3221890\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q109420045\n",
+      "Birthday not found on FIDE for FIDE id 2200821\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q11701826\n",
+      "Birthday not found on FIDE for FIDE id 14000172\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q104760479\n",
+      "Birthday not found on FIDE for FIDE id 4127382\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q80695681\n",
+      "Birthday not found on FIDE for FIDE id 4103106\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q101245497\n",
+      "Birthday not found on FIDE for FIDE id 4142861\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q101541241\n",
+      "Birthday not found on FIDE for FIDE id 4104463\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q101593027\n",
+      "Birthday not found on FIDE for FIDE id 4104684\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q101780469\n",
+      "Birthday not found on FIDE for FIDE id 150098007\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q115673943\n",
+      "Birthday not found on FIDE for FIDE id 4103068\n",
+      "WIKIDATA: http://www.wikidata.org/entity/Q101784318\n"
+     ]
+    }
+   ],
+   "source": [
+    "count = 0\n",
+    "f = open(\"invalid_fide.txt\", 'w')\n",
+    "lst = []\n",
+    "for e in wikidata_players_list:\n",
+    "    try:\n",
+    "        if (e['dateOfBirth'][:4] != fide_dict[e['fideID']]['birthday']):\n",
+    "            \n",
+    "            if (fide_dict[e['fideID']]['birthday'] == None):\n",
+    "                print(\"Birthday not found on FIDE for FIDE id %s\" % e['fideID'])\n",
+    "                print(\"WIKIDATA: %s\" % e['item'])\n",
+    "                continue\n",
+    "            external_url = 'https://ratings.fide.com/profile/' + e['fideID']\n",
+    "            row = {'item_id': e['item'][31:], 'property_id': 'P569', 'statement_guid': e['dobStatementGUID'], 'wikidata_value': e['dateOfBirth'], 'meta_wikidata_value': 'Q1985727', 'external_value': fide_dict[e['fideID']]['birthday'], 'external_url': external_url, 'type':'statement'}\n",
```

Comment:
This line is a little long, could you split it into multiple lines such that each key/column is its own line?